pages/ButtonsAndLinks.js

```diff
@@ -4,45 +4,108 @@ import linkStyles from '../css/link.css';
 import ButtonDiagram from './assets/button-diagram.svg';
 import buttonStyles from '../css/button.css';
 import styles from './css/ButtonPage.css';
+import { MenuItem } from './components/MenuItem';
 
 function ButtonExample({ label, colour, cssClass, blurb }) {
   return (
     <div className={styles['button-section']}>
-      <h4>
-        {label} ({colour})
-      </h4>
+      <h4>{label}</h4>
+      <div>{blurb}</div>
+      <br />
       <div>
-        CSS class: <code>{cssClass}</code>
+        CSS class: &nbsp;<code>{cssClass}</code>
```

Comment:
Same here about the `&nbsp;`

This function/component is duplicated with the above file, so it would be better ot move it into its own file, and import it

Reply:
You haven't done this

Reply:
@Amwam I _think_ I have now